public_website/src/ui/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -1,76 +1,101 @@
-import React from 'react'
+import React, { useContext } from 'react'
 import Link from 'next/link'
-import { useRouter } from 'next/router'
+import { usePathname, useRouter } from 'next/navigation'
 import styled, { css } from 'styled-components'
 
 import { ChevronDown } from '../icons/ChevronDown'
-type LinkType = { Label: string; URL: string }[]
+import { BreadcrumbContext } from './breadcrumb-context'
 
-type Breadcrumb = {
-  id: number
-  fils: LinkType
-  parent: { id: number; Label: string; URL: string }
+interface Breadcrumb2Props {
+  isUnderHeader?: boolean
+  className?: string
 }
```

Comment:
Possible sans le "2" ? 
```suggestion
interface Breadcrumb2Props {
  isUnderHeader?: boolean
  className?: string
}
```